src/fileloader.h

```diff
@@ -19,129 +19,134 @@ struct Node
 	ContentIt propsBegin;
 	ContentIt propsEnd;
 	uint8_t type;
-	enum NodeChar: uint8_t
+	enum NodeChar : uint8_t
 	{
 		ESCAPE = 0xFD,
 		START = 0xFE,
 		END = 0xFF,
 	};
 };
 
-struct LoadError : std::exception {
+struct LoadError : std::exception
+{
 	const char* what() const noexcept override = 0;
 };
 
-struct InvalidOTBFormat final : LoadError {
-	const char* what() const noexcept override {
-		return "Invalid OTBM file format";
-	}
+struct InvalidOTBFormat final : LoadError
+{
+	const char* what() const noexcept override { return "Invalid OTBM file format"; }
 };
 
-class Loader {
+class Loader
+{
 	MappedFile fileContents;
 	Node root;
 	std::vector<char> propBuffer;
+
 public:
 	Loader(const std::string& fileName, const Identifier& acceptedIdentifier);
 	bool getProps(const Node& node, PropStream& props);
 	const Node& parseTree();
 };
 
-} //namespace OTB
+} // namespace OTB
 
 class PropStream
 {
-	public:
-		void init(const char* a, size_t size) {
-			p = a;
-			end = a + size;
-		}
+public:
+	void init(const char* a, size_t size)
+	{
+		p = a;
+		end = a + size;
+	}
+
+	size_t size() const { return end - p; }
 
-		size_t size() const {
-			return end - p;
+	template <typename T>
+	bool read(T& ret)
+	{
+		if (size() < sizeof(T)) {
+			return false;
 		}
 
-		template <typename T>
-		bool read(T& ret) {
-			if (size() < sizeof(T)) {
-				return false;
-			}
+		memcpy(&ret, p, sizeof(T));
+		p += sizeof(T);
+		return true;
+	}
 
-			memcpy(&ret, p, sizeof(T));
-			p += sizeof(T);
-			return true;
+	bool readString(std::string& ret)
+	{
+		uint16_t strLen;
+		if (!read<uint16_t>(strLen)) {
+			return false;
 		}
 
-		bool readString(std::string& ret) {
-			uint16_t strLen;
-			if (!read<uint16_t>(strLen)) {
-				return false;
-			}
-
-			if (size() < strLen) {
-				return false;
-			}
-
-			char* str = new char[strLen + 1];
-			memcpy(str, p, strLen);
-			str[strLen] = 0;
-			ret.assign(str, strLen);
-			delete[] str;
-			p += strLen;
-			return true;
+		if (size() < strLen) {
+			return false;
 		}
 
-		bool skip(size_t n) {
-			if (size() < n) {
-				return false;
-			}
+		char* str = new char[strLen + 1];
+		memcpy(str, p, strLen);
+		str[strLen] = 0;
+		ret.assign(str, strLen);
+		delete[] str;
+		p += strLen;
+		return true;
+	}
```

Comment:
Should return `std::pair<std::string, bool>` instead of passing return as argument

Reply:
I will fix this in another PR.